roles/openshift_logging/tasks/main.yaml

```diff
@@ -24,13 +24,44 @@
   check_mode: no
   become: false
 
+- oc_obj:
+    state: list
+    kind: dc
+    all_namespaces: true
+    selector: "logging-infra,provider=openshift"
+  register: _logging_dcs
+
+- assert:
+    that:
+    - _logging_dcs.results.results[0]['items'] | map(attribute='metadata.namespace') | list | unique | count <= 1
+    msg: "Found aggregated logging deploymentconfigs  in multiple namespaces which is not supported"
```

Comment:
minor nit, but can you remove the extra space between "deploymentconfigs" and "in" ?